pkg/sql/schemachanger/sctest/cumulative.go

```diff
@@ -52,34 +481,66 @@ import (
 func cumulativeTest(
```

Comment:
can you add some comment about how we use two-pass to handle the introduction of `exec-stage` and `exec-query` here?

Reply:
Added a comment here:
```
// To support injection of statements and rewriting the cumulative function
// will through the file twice using the data driven functions:
//
//  1. The first pass will collect all the stage-exec/stage-query statements,
//     setup commands. Then executing the test statement and setup statements
//     via the tf callback. The callback can if the rewrite is enabled update
//     the expected output inside stageExecStmtMap (see stageExecStmt.Exec).
//
//  2. The second pass will for any stage-exec/stage-query functions look up
//     the output using stageExecStmtMap.GetExpectedOutputForPos and return
//     it, only when rewrite is enabled.
```

Reply:
nit: "... will go through ..."